### PR TITLE
[INLONG-10434][Manager] Allow creating cluster nodes without setting port

### LINKED ISSUE #10434: [Improve][Manager] Allow creating cluster nodes without setting port
### Description

Allow creating cluster nodes without setting port.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
Fixes #10434

### Motivation

Allow creating cluster nodes without setting port.
At present, the agent node does not have the concept of port, so there is no need to set port information.
### Modifications

Allow creating cluster nodes without setting port.

### CODE REVIEW

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/heartbeat/HeartbeatManager.java [hunk 1]
```diff
@@ -274,11 +281,13 @@ private void evictClusterNode(HeartbeatMsg heartbeat) {
...
         }
 
-        for (int i = 0; i < ports.length; i++) {
+        for (int i = 0; i < ips.length; i++) {
             // deep clone the heartbeat
             HeartbeatMsg heartbeatMsg = JsonUtils.parseObject(JsonUtils.toJsonByte(heartbeat), HeartbeatMsg.class);
             assert heartbeatMsg != null;
-            heartbeatMsg.setPort(ports[i].trim());
+            if (StringUtils.isNotBlank(heartbeat.getPort())) {
+                heartbeatMsg.setPort(ports[i].trim());
```
Comment (aloyszhang) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/heartbeat/HeartbeatManager.java:289]: `ports` may be null even if `heartbeat.getPort()` is not blank.

Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/heartbeat/HeartbeatManager.java:289]: ports is not null when heartbeat.getPort() is not blank.

Comment (aloyszhang) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/heartbeat/HeartbeatManager.java:289]: `ports` is set to null when `ports.length < ips.length`

Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/heartbeat/HeartbeatManager.java:289]: Fixed.

**Review by aloyszhang — APPROVED** [review]

**Review by vernedeng — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**